web-common-marlowe/src/Marlowe/Market/Contract1.purs

```diff
@@ -18,185 +18,123 @@ contractTemplate = { metaData, extendedContract }
 metaData :: MetaData
 metaData =
   { contractType: Escrow
-  , contractName: "Escrow"
-  , contractDescription: "Escrow is a financial arrangement where a third party holds and regulates payment of the funds required for two parties involved in a given transaction."
+  , contractName: "Simple escrow"
+  , contractDescription: "Regulates a money exchange between a \"Buyer\" and a \"Seller\". If there is a disagreement, an \"Arbiter\" will decide whether the money is refunded or paid to the \"Seller\"."
   , roleDescriptions:
       fromFoldable
-        [ "alice" /\ "about the alice role"
-        , "bob" /\ "about the bob role"
-        , "carol" /\ "about the carol role"
+        [ "Arbiter" /\ "The party that will choose who gets the money in the event of a disagreement between the \"Buyer\" and the \"Seller\" about the outcome."
+        , "Buyer" /\ "The party that wants to buy the item. Payment is made to the seller if they acknowledge receiving the item. "
+        , "Seller" /\ "The party that wants to sell the item. They receive the payment if the exchange is uneventful."
         ]
   , slotParameterDescriptions:
       fromFoldable
-        [ "aliceTimeout" /\ "about the aliceTimeout"
-        , "arbitrageTimeout" /\ "about the arbitrageTimeout"
-        , "bobTimeout" /\ "about the bobTimeout"
-        , "depositSlot" /\ "about the depositSlot"
+        [ "Buyer's deposit timeout" /\ "Deadline by which the \"Buyer\" must deposit the selling  \"Price\" in the contract."
```

Comment:
@palas , in a future PR we should remove all Slot Params, right?

Reply:
We removed some of them, but when they are few they don't look bad. It is up to you, we can have them in the playground for example